app/webpacker/css/admin/v2/colors.scss

```diff
@@ -0,0 +1,7 @@
+$orange-v2: #F27052;
+$orange-v2-light: #F5947D;
+$orange-v2-lighter: #F8B7A8;
+$orange-v2-lightest: #FCDBD4;
+
+$dark-grey-v2: #333333;
+$medium-dark-grey-v2: #444444;
```

Comment:
I like that you've included 'v2' to avoid conflicts. This is then effectively a namespace, so I think it would be better to have this at the start of the name (eg `$v2-orange`).

Reply:
Yep, I agree, will do that.